cmd/sync/aws.go

```diff
@@ -69,27 +76,29 @@ func (client *AWSClient) configure() error {
 	return nil
 }
 
-// validateAndSaveConfig parses and validates AWSClient config and saves it
-func (client *AWSClient) validateAndSaveConfig(data []byte) error {
+// parseConfig parses and validates AWSClient config
+func (client *AWSClient) parseConfig(data []byte) (*awsConfig, error) {
```

Comment:
if  `client` is not used, perhaps it makes sense to convert this method to a function?